### PR TITLE
Adding solutions 2 to 12 for Assignment 3

### PR DESCRIPTION
**Pull request by ShitalSDec19**

### CODE REVIEW

File: src/shitalSDec19/assignment3/Solution1.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+package shitalSDec19.assignment3;
+import java.util.Scanner;
+public class Solution1 {
+	
+	void findPositiveNos(int [] n)
+	{		
+		int arr1[] = new int[n.length];
+		int arr2[] = new int[n.length];
+		
+		for (int i =0; i<n.length; i++)
```
Comment (harshvegada) [inline src/shitalSDec19/assignment3/Solution1.java]: I didn't get this logic.
could you explain me in class

File: src/shitalSDec19/assignment3/Solution13.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	
+	
+	public static void main(String[] args) {
+		System.out.println("please provide no of iterations you want");
+		 Scanner sc = new Scanner (System.in);
+		int size = sc.nextInt();
+		for (int i=0; i<size; i++)
+			{
+				System.out.println("Enter number");
+				Scanner scan = new Scanner (System.in);
```
Comment (harshvegada) [inline src/shitalSDec19/assignment3/Solution13.java]: why we use two time scanner class in same method.
please give explanation. @ShitalSDec19

File: src/shitalSDec19/assignment3/Solution15.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+public class Solution15 {
+
+	static String[] output = {};
+	public static void main(String[] args) {
+		String [] input = {"krishna", "maulik", "aakash", "harsh", "aakansha", "abhishek"};
+		for (int i = 0; i<input.length-2; i++)
+		{
+			String s = input[i];
+			String [] a = s.split(",");
+			if (input.length > (a[i].length()));
```
Comment (harshvegada) [inline src/shitalSDec19/assignment3/Solution15.java]: You not get exact question, I said print those name only whose length is greater than 7

File: src/shitalSDec19/assignment3/Solution6.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	
+	void findMaxLengthString(String [] arr)
+	{ 
+		String b="";
+		String reverse = "";
+		int maxLength=0;
+		for (int i=0; i<arr.length; i++)
+		{
+			String s = arr[i];
+			String [] a = s.split(",");
```
Comment (harshvegada) [inline src/shitalSDec19/assignment3/Solution6.java]: what is use of split over here?

**Comment by harshvegada** [comment]
Do changes as per comment and raise PR again

File: src/shitalSDec19/assignment3/Solution13.java (hunk 2 above)
Comment (ShitalSDec19) [reply src/shitalSDec19/assignment3/Solution13.java]: i have updated my program, please review.

File: src/shitalSDec19/assignment3/Solution11.java [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+	{
+		int original = n;
+		int i=0;
+		boolean flag = false;
+		while(n!=0)
+			{
+			reverse = reverse*10;
+			reverse = reverse + n%10;
+			n = n/10;
+			if (original == reverse)
```
[the remark this answers is not in the document]
Comment (ShitalSDec19) [reply src/shitalSDec19/assignment3/Solution11.java]: I have update my program. Please review.

File: src/shitalSDec19/assignment3/Solution6.java (hunk 4 above)
Comment (ShitalSDec19) [reply src/shitalSDec19/assignment3/Solution6.java]: I was trying to make use of split but it was of no use. I have updated my program but still i am unable to print expected reverse string. Please review.

File: src/shitalSDec19/assignment3/Solution14.java [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+		if (arr1.length == arr2.length)
+		{
+			for (int i = 0; i<arr1.length; i++)
+			{
+				for (int j=0; j<arr2.length; j++)
+				{
+					if (arr1[i] != arr2[j])
+						
+				arr3 = arr2;
+					System.out.println(arr3[i]);
```
[the remark this answers is not in the document]
Comment (ShitalSDec19) [reply src/shitalSDec19/assignment3/Solution14.java]: I was unable to save array output to array, hence i was doing some R&D. Now that i have solved solution11 from 2nd assignment and i learnt how to save output in array. I have updated my program, please review.

File: src/shitalSDec19/assignment3/Solution12.java [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+import java.util.Scanner;
+public class Solution12 {
+	String reverse="";
+	boolean flag = false;
+	void doReverse(String s)
+	{
+		for (int i=s.length()-1; i>=0;i--)
+		{
+			reverse = reverse + s.charAt(i);
+			if (s.equals(reverse))
```
[the remark this answers is not in the document]
Comment (ShitalSDec19) [reply src/shitalSDec19/assignment3/Solution12.java]: I have updated my program, please review.

**Pull request merged**